bot_injected.Dockerfile

```diff
@@ -15,9 +19,12 @@ LABEL io.parity.image.authors="cicd-team@parity.io" \
     io.parity.image.created="${BUILD_DATE}"
 
 WORKDIR /bot
+
+COPY ./package.json .
+COPY ./yarn.lock .
```

Comment:
You can move this to a single line
```suggestion
COPY ./package.json ./yarn.lock .
```

Reply:
+1, otherwise it'd create a separate layer for each copy command

Reply:
Done ✅